examples/soap-calculator/README.md

```diff
@@ -8,7 +8,7 @@ This example project shows how to integrate a SOAP web service with LoopBack 4
 and expose its methods through the REST API server. Acceptance and Integration
 tests are provided.
 
-![soap-calculator-overview](../../docs/img/loopback-example-soap-calculator_figure1.png)
+![soap-calculator-overview](https://loopback.io/pages/en/lb4/imgs/loopback-example-soap-calculator_figure1.png)
```

Comment:
Hmm, I wish relative path for images could work so that we can preview the diagrams on github.

Reply:
I just realized that we reference an image in `docs`.

Reply:
Since we have the full tutorial at https://loopback.io/doc/en/lb4/soap-calculator-tutorial.html, can we just remove the content here from the README and simply have a link to the tutorial?

Reply:
OK. let me do this change.

Reply:
IMO `soap-calculator-tutorial` should be pulling in and displaying this README instead of having separate content as it keeps the two pages in sync. This is the approach used by `TodoList` tutorial and `Todo` tutorial.

See:
https://raw.githubusercontent.com/strongloop/loopback-next/master/docs/site/todo-tutorial.md

as an example of how to pull in the README.md

Reply:
Yes @virkt25  , I will follow that direction, to standardize  it.